Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -412,12 +437,12 @@ void LoadBankFromDiskTask::run() {
 
   } // try block
   catch (std::exception &e) {
-    m_loader.alg->getLogger().error()
+    m_loader.m_loadAlgorithm->getLogger().error()
         << "Error while loading bank " << entry_name << ":\n";
-    m_loader.alg->getLogger().error() << e.what() << '\n';
+    m_loader.m_loadAlgorithm->getLogger().error() << e.what() << '\n';
     m_loadError = true;
   } catch (...) {
```

Comment:
This is a bit of a segway, but is there a possibility of the nexus api propagating H5 exceptions? In which case falls into (...). Note that in H5cpp, H5Exception does not have std::exception as base https://support.hdfgroup.org/HDF5/doc/cpplus_RM/_h5_exception_8h_source.html

Reply:
The SNS have plans to rewrite this using H5 consistently. May not be worth the effort.